### PR TITLE
Lycophron Skeleton and first commands

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
```
Comment (alejandromumo) [inline pyproject.toml:1]: Are we using poetry?

File: lycophron/project.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            else:
+                os.mkdir(self.name)
+
+            click.secho("Folder Created!", fg="green")
+        except FileExistsError:
+            click.secho("The folder already exists!", fg="green")
+
+    def initialize(self):
+        """Initialize the project"""
+        self.create_folder()
```
Comment (alejandromumo) [inline lycophron/project.py:64]: Why is it creating two folders?

File: lycophron/models.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+session = sessionmaker()
+session.configure(bind=engine)
+
+Model = declarative_base()
+
+
+class Record(Model):
+    __tablename__ = "record"
+    id = Column(Integer, primary_key=True)
```
Comment (alejandromumo) [inline lycophron/models.py:15]: TODO: use an updated model

File: lycophron/project.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+    def create_db(self):
+        """Create a database"""
+        models.create_all()
+        click.secho("DB Created!", fg="green")
+
+    def create_csv(self):
+        """Create a CSV"""
+        header = [
+            "id",
```
Comment (alejandromumo) [inline lycophron/project.py:26]: TODO: avoid hardcoded field names

**Comment by slint** [comment]
Closed in favor of #16

**Pull request closed without merging**